packages/agreement-process/test/db.test.ts

```diff
@@ -79,6 +93,11 @@ describe("database test", async () => {
     await postgresDB.none("TRUNCATE TABLE catalog.events RESTART IDENTITY");
   });
 
+  afterAll(async () => {
+    await postgreSqlContainer.stop();
+    await mongodbContainer.stop();
+  });
```

Comment:
In my opinion this could enforce to stopping containers between different  package's integration tests, so I leave this changes.

Reply:
In my opinion it is not required. Each test suite runs concurrently, and the current `testcontainers` setup ensures that each test suite creates/uses/stops/deletes its own containers, so there is no need to manually stop them.
Currently (with the code correctly refactored) I am not able to replicate a situation where the containers are not correctly stopped: I can investigate more if it happens later on.